variants/backend-base/config/environments/test.rb

```diff
@@ -1,7 +1,3 @@
-gsub_file "config/environments/test.rb",
-          "config.eager_load = false",
-          "config.eager_load = defined?(SimpleCov).present?"
```

Comment:
note: Rails 7.1 has changed this to `ENV["CI"].present?` rather than `true`

I don't think it's better to check if `SimpleCov` is defined but maybe I'm wrong?